common/src/main/resources/ftbultimine-common.mixins.json

```diff
@@ -11,5 +11,6 @@
   "injectors": {
     "defaultRequire": 1
   },
-  "minVersion": "0.8"
+  "minVersion": "0.8",
+  "refmap": "ftb-ultimine-common-refmap.json"
```

Comment:
What is the purpose of this change?  With it present, the instance fails to start for me with a mixin-related exception. Removing it allowed the instance to start.

Reply:
huh... not sure, it's been working for me :/ I'll look into it. Good catch.

Not entirely sure why it's there 😆

Reply:
Resolved in https://github.com/FTBTeam/FTB-Ultimine/pull/151/commits/de13ac21eb5f8e83e4ccd94ff2d7ee6f1264a8ea